src/modules/assessments/components/AssessmentForm.tsx

```diff
@@ -422,26 +436,24 @@ const AssessmentForm: React.FC<Props> = ({
             warnIfEmpty={!!assessment || hasAnyValue(errors)}
           />
         )}
-      </Grid>
+      </FormNavigationContainer>
 
       {/* Dialog for selecting autofill record */}
-      {definition && (
-        <RecordPickerDialog
-          id='assessmentPickerDialog'
-          clientId={client.id}
-          open={dialogOpen}
-          role={definition.role}
-          onSelected={onSelectAutofillRecord}
-          onCancel={() => setDialogOpen(false)}
-          description={
-            <Typography variant='body2' sx={{ my: 2 }}>
-              Select a previous assessment to populate the current assessment.
-              Any changes you have made will be overwritten.
-            </Typography>
-          }
-        />
-      )}
-    </Grid>
+      <RecordPickerDialog
```

Comment:
This is unchanged, I just removed `definition &&` since `definition` is always present -- it's either `viewingDefinition` or `editingDefinition`, both of which are required props.